liferay-echo/liferay-portal
# LPS-193741 Migrate frontend-taglib sidebar-panel AUI file to javascript

Motivation
=======
The motivation behind this ticket is to migrate from AUI old javascript files to functional/react js components.
https://liferay.atlassian.net/browse/LPS-193741

Proposed Solution
========
What I did to solve this problem is to change the file structure and to use equivalents methods in new js structure

Steps to Verify
========
1. Go to Content & Data -> Documents & Media -> go to provided by liferay folder
2. Open the SidebarPanel by clicking on the info button
3. Check that the panel is rendered and functions correctly
